src/browser/modules/Stream/CypherFrame/ErrorsView/MissingParamsTemplateLink.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+
+import {
+  StyledMissingParamsTemplateLink,
+  StyledParamsTemplateClickableArea,
+  StyledSpecifyParamsText
+} from 'project-root/src/browser/modules/Stream/styled'
+import { stringModifier } from 'services/bolt/cypherTypesFormatting'
+import { BrowserError } from 'services/exceptions'
+import { stringifyMod } from 'services/utils'
+
+type MissingParamsTemplateLinkProps = {
+  error: BrowserError
+  params: Record<string, unknown>
+  onSetFrameCmd: (cmd: string, autoExec: boolean) => void
+}
+export const MissingParamsTemplateLink = ({
+  onSetFrameCmd,
+  params,
+  error
+}: MissingParamsTemplateLinkProps) => {
+  const onGenerateMissingParamsTemplate = (
```

Comment:
A pet peeve of mine is functions starting with "on" that don't refer to an event 😅 It's fine leaving it as is, but perhaps we could call it either in the `onMissingParamsMessageClick` or `generateMissingParamsTemplate`